src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/PtvTranslator.java

```diff
@@ -124,7 +110,15 @@ public OrganizationService translateOrganizationService(OrganizationServiceId ku
     return result;
   }
 
-  public Service translateService(ServiceId serviceKuntaApiId, fi.otavanopisto.restfulptv.client.model.Service ptvService, StatutoryDescription ptvStatutoryDescription) {
+  public Service translateService(ServiceId serviceKuntaApiId, 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Method has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00107)